packages/runtime/container-runtime/api-report/container-runtime.legacy.alpha.api.md

```diff
@@ -354,6 +359,16 @@ export interface IOnDemandSummarizeOptions extends ISummarizeOptions {
     readonly retryOnFailure?: boolean;
 }
 
+// @alpha
+export interface IProtocolOptions {
+    readonly compressionOptions?: ICompressionRuntimeOptions;
+    readonly enableGCSweep?: true | undefined;
```

Comment:
`gcOptions` is a tricky case, since it contains a compatability/protocol related option. Ultimately, I decided to just include that specific option (`enableGCSweep`) in `IProtocolOptions` and leave the rest of in `IContainerRuntimeOptions`.